vicare_exporter/metrics.py

```diff
@@ -120,7 +118,7 @@ def _fetch_devices_features(vicare: PyViCare) -> int:
 
     n_features = 0
     for device in vicare.devices:
-        features = device.service.fetch_all_features()
+        features = device.asGazBoiler().service.fetch_all_features()
```

Comment:
As we're only using the `service`  later on, does this have any effect on the results?